src/components/formikComponents/formikDropdown.js

```diff
@@ -0,0 +1,435 @@
+import React, { useRef, useState } from 'react'
+import PropTypes from 'prop-types'
+import styled, { css } from 'styled-components'
+import Keycodes from 'common/keycodes'
+import { useOutsideClick } from 'components/hooks/outside-click'
+import Chevron from 'images/svg/chevron-bottom.svg'
+import device from 'themes/device'
+import { Flex } from 'components/containers'
+import { Text } from 'components/elements'
+// SVG Component
+// import CrossIcon from 'images/svg/cross.svg'
+
+const Symbol = styled(Flex)`
+    width: fit-content;
+    margin-top: 8px;
+
+    svg {
+        width: 32px;
+        height: 32px;
+        margin-right: 0.8rem;
+    }
+    ${Text} {
+        font-weight: normal;
+        font-size: var(--text-size-xs);
+        line-height: 1.14;
+        margin-top: 8px;
+
+        @media ${device.mobileL} {
+            font-size: 14px;
+        }
+    }
+`
+
+const DropdownContainer = styled.ul`
+    list-style: none;
+    position: relative;
+    border: 1px solid var(--color-grey-7);
+    cursor: pointer;
+    padding: 0;
+    border-radius: 4px;
+    height: 40px;
+    margin-bottom: 0;
+
+    /* ul has no focus attributes, it needs to pass on active props instead */
+    ${(props) => props.active && 'border-color: var(--color-green) !important;'}
+
+    &:hover {
+        border-color: var(--color-grey-5);
+    }
+
+    ${(props) =>
+        props.has_short_name &&
+        css`
+            width: auto;
+            min-width: 6.5rem;
+
+            @media ${device.tabletL} {
+                top: inherit;
+                width: auto;
+                min-width: 8rem;
+            }
+            @media ${device.mobileL} {
+                min-width: 7rem;
+            }
+        `}
+
+    ${(props) =>
+        !props.error &&
+        css`
+            border-color: var(--color-grey-7);
+        `}
+
+    ${(props) =>
+        props.error &&
+        css`
+            border-color: var(--color-red-1) !important;
```

Comment:
why are we using important here?

Reply:
I copied this styling from the `input.js` should we just remove it ?

Reply:
I tried removing the important but it gives me  Cannot parse selector error